Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionsTreeView.cs

```diff
@@ -619,7 +616,7 @@ public static List<TreeViewItemData<ActionOrBindingData>> GetActionsAsTreeViewDa
                                 if (isVisible)
                                 {
                                     var name = GetHumanReadableCompositeName(nextBinding, state.selectedControlScheme, controlSchemes);
-                                    compositeItems.Add(new TreeViewItemData<ActionOrBindingData>(GetIdForGuid(new Guid(nextBinding.id), idDictionary),
+                                    compositeItems.Add(new TreeViewItemData<ActionOrBindingData>(treeviewItemIDCounter++,
```

Comment:
Not sure I understand this binding counter, I though items where mapped to the asset GUIDs for stability?

Reply:
This deterministic counter replaces the idDictionary, which appeared to be non-deterministic. The idDictionary is not used anywhere else.

Reply:
I think this was written this way to maintain the same Id's for actions/bindings over rebuilds. This condition is not met with this change. I don't remember the particularites why it is solved this way, it might be that there is no urge to do that anymore, but I believe there was a reason to maintain it (at least back then).